.github/dependabot.yml

```diff
@@ -23,11 +23,3 @@ updates:
     interval: daily
     time: "03:00"
   open-pull-requests-limit: 10
-- package-ecosystem: npm
-  directory: "/modules/ytp-assets-common/"
-  schedule:
-    interval: daily
-    time: "03:00"
-  open-pull-requests-limit: 10
-  registries:
-  - npm-registry-npm-fontawesome-com
```

Comment:
Why this was removed ? It should be checking if there are updates to font awesome.

Reply:
@Zharktas I'm just going with the assumption that modules directory will be deleted from this repository when everything regarding this docker migration is done.

So shouldn't the dependabot check should be added to opendata-drupal and opendata-ckan repositories instead? Also I remember discussing with @bzar that later with the ckan 2.9 migration, we should probably have ytp-assets-common as a submodule.

Reply:
ah, true. Then this is not an issue.